nilearn/experimental/surface/_datasets.py

```diff
@@ -0,0 +1,61 @@
+"""Fetching a few example datasets to use during development.
+
+eventually nilearn.datasets would be updated
+"""
+from typing import Dict, Sequence, Tuple
+
+from nilearn import datasets
+from nilearn.experimental.surface import _io
+from nilearn.experimental.surface._surface_image import (
+    Mesh,
+    PolyMesh,
+    SurfaceImage,
+)
+
+
+def load_fsaverage(mesh_name: str = "fsaverage5") -> Dict[str, PolyMesh]:
```

Comment:
right, I remember we thought about that. but I wonder if it will result in user having to call that function several times, as it is quite frequent to need 2 or 3 of them -- eg pial and white matter for volume to surface projection, then inflated for plotting? but at the same time when only 1 is needed adding the argument would make it faster